jax/_src/numpy/fft.py

```diff
@@ -183,10 +183,71 @@ def fftn(a: ArrayLike, s: Shape | None = None,
   return _fft_core('fftn', xla_client.FftType.FFT, a, s, axes, norm)
 
 
-@implements(np.fft.ifftn)
 def ifftn(a: ArrayLike, s: Shape | None = None,
           axes: Sequence[int] | None = None,
           norm: str | None = None) -> Array:
+  r"""Compute inverse discrete Fourier transform of multidimensional array along
+  given axes.
+
+  JAX implementation of :func:`numpy.fft.ifftn`.
+
+  Args:
+    a: input array
+    s: sequence of integers. Specifies the shape of the result. If not specified,
+      it will default to the shape of ``a`` along the specified ``axes``.
+    axes: sequence of integers, default=None. Specifies the axes along which the
+      transform is computed.
```

Comment:
We should mention the meaning of None here rather than in the example description.

Reply:
Mentioned the meaning of None at the description of axes.